spec/api/channel_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'discordrb'
+
+describe Discordrb::API::Channel do
+  let(:token) { double('token') }
+  let(:channel_id) { instance_double(String, 'channel_id') }
+
+  before do
+    allow(token).to receive(:to_s).and_return('bot_token')
+    allow(channel_id).to receive(:to_s).and_return('channel_id')
+  end
```

Comment:
We can move these method stubs inside the double initialiser
```suggestion
  let(:token) { double('token', to_s: 'bot_token') }
  let(:channel_id) { instance_double(String, 'channel_id', to_s: 'channel_id') }
```